assets/js/components/UserMenu.stories.js

```diff
@@ -1,7 +1,7 @@
 /**
  * UserMenu stories.
  *
- * Site Kit by Google, Copyright 2021 Google LLC
+ * Site Kit by Google, Copyright 2022 Google LLC
```

Comment:
Usually we don't update the copyright of existing files, and given this was renamed it probably makes sense to leave it as-is.

```suggestion
 * Site Kit by Google, Copyright 2021 Google LLC
```